examples/multimodal/multimodal_llm/neva/eval/mixtral_eval.py

```diff
@@ -0,0 +1,232 @@
+import argparse
+import json
+import math
+import os
+from collections import defaultdict
+
+import numpy as np
+import requests
+import shortuuid
+import torch
+from PIL import Image
+from tqdm import tqdm
+
+invoke_url = "https://api.nvcf.nvidia.com/v2/nvcf/pexec/functions/8f4118ba-60a8-4e6b-8574-e38a4067a4a3"
+
+API_TOKEN = ""  # ADD NGC API TOKEN HERE
```

Comment:
usually it's better to pass it as environment variable. so user can do
```
API_TOKEN=xxx python xx.py
```

Reply:
Thanks for the tip!